src/averageAges.js

```diff
@@ -37,8 +56,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
-}
+  const woman = people.filter(person =>
```

Comment:
Grammar mistake
Check @dinaminahi comment
```suggestion
  const women = people.filter(person =>
```